js/enquiry.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function(){
+	if($('#enquiry-form').show()){
+		$('.fakeUpload').on('click',function (){
+			$('#file').click();
+		});
+		$('#file').on('change',function(){
+			var filename = this.value;
+		    var lastIndex = filename.lastIndexOf("\\");
+		    if (lastIndex >= 0)
+		        filename = filename.substring(lastIndex + 1);
+	    	$('#filename').html(filename);
+		});
+		 $('.submitForm').on('click', function(){
+		 	var checkedAtLeastOne = false;
+			$('input[type="checkbox"]').each(function() {
+			    if ($(this).is(":checked")) 
+			        checkedAtLeastOne = true;
+			});
+			if(!checkedAtLeastOne)
+				$('#checkbox1').prop("checked",true);
```

Comment:
If user hasn't select the services type, we should not automatically select the first one. It will make our data incorrect. I think we could leave it blank or show user message to encourage them to choose one.